src/Features/CSharp/Portable/Completion/CompletionProviders/DeclarationNameCompletionProvider.DeclarationInfo.cs

```diff
@@ -100,7 +100,7 @@ internal struct NameDeclarationInfo
                         semanticModel,
                         GetNodeDenotingTheTypeOfTupleArgument,
                         _ => default(SyntaxTokenList),
-                        _ => ImmutableArray.Create(SymbolKind.Local), cancellationToken);
+                        _ => ImmutableArray.Create(new SymbolKindOrTypeKind(SymbolKind.Local)), cancellationToken);
```

Comment:
This is technically wrong. User's preferences for locals shouldn't apply to tuple name suggestions. This needs to be fixed as soon as naming preferences for tuples are implemented.